tensorboard/pip_package/setup.py

```diff
@@ -66,6 +65,8 @@ def get_readme():
             'webfiles.zip',
         ],
     },
+    # Disallow python 3.0 and 3.1 which lack a 'futures' module (see above).
```

Comment:
Whelp, I guess 3.0 and 3.1 users are out of luck in terms of installing TensorBoard, but I also see no way around this if we are to use features from the `futures` (or `concurrent.futures`) module. Makes sense.

Reply:
Yeah, see more discussion on #916 where Justine noted that TF itself only officially supports python 2.7 and 3.4+.